johnsonandjohnson/Bodiless-JS
# feat(fclasses): Change startWith so that it does not replace the whol…

…e item but instead just the starting Component





## Changes

Change the functionality of startWith, to not replace the component where it is but to change the component that apply design is going to start with.

## Test Instructions


Use startWith after applying HOC and see that the component is changed but the HOC is still applied.

## Related Issues



BREAKING CHANGE: startWith functionality is changed
